Tests/NIOHTTP1Tests/HTTPServerUpgradeTests.swift

```diff
@@ -38,7 +40,7 @@ extension ChannelPipeline {
     fileprivate func assertContainsUpgrader() {
         #if !canImport(Darwin) || swift(>=5.10)
         do {
-            _ = try self.context(handlerType: NIOTypedHTTPServerUpgradeHandler<Bool>.self).wait()
+            _ = try self.context(handlerType: NIOTypedHTTPServerUpgradeHandler<Bool>.self).map { _ in () }.wait()
```

Comment:
This pattern is common enough that you might get more mileage to by factoring this out to a `ChannelPipeline.contains` function.